src/style.css

```diff
@@ -0,0 +1,103 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  --header__height: 60px;
+  --item__size: 15px;
+  --line__size: 4px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  min-height: var(--header__height);
+  align-items: center;
+  /*background-color: gray;*/
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  border-radius: 8px;
+}
+
+.header:focus,
+.header:hover {
+  outline: 1px solid #00acdc;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  align-self: stretch;
+  margin-right: 50px;
+  /*background-color: yellow;*/
+}
+
+.menu { 
+  /*position: relative;*/
+  min-height: var(--header__height);
+  margin-top: 0;
+  margin-bottom: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 20px;
+  flex-grow: 1;
+  /*background-color: rgb(216, 157, 115);*/
+}
+
+.menu_li {
+  /*min-height: inherit;*/
+  list-style: none;
+  /*background-color: green;*/
```

Comment:
```suggestion
```
Property `list-style` should be applied to the list (your `.menu` element) but not for the list item. So write it there.
Also, let's give 'display:flex' for this element and 'position: relative' to be able to position your `:after` element to this element.
And as I've written earlier, it's better to give margins here to make the horizontal spacing between list items (`margin-right: 20px;). But last element should not have the right margin, so you may use `.menu_li:last-child` to reset right margin for the last element